### PR TITLE
Amber/creating sdk for Conversations authoring

### PR DESCRIPTION
**Pull request by amber-ccc**
…tStatusResponse

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by bidisha-c — COMMENTED** [review]
Change log is missing.

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+Install the client library for .NET with [NuGet](https://www.nuget.org/ ):
+
+```dotnetcli
+dotnet add package Azure.AI.Language.Conversations.Authoring --prerelease
+```
+
+|SDK version  |Supported API version of service
+|-------------|-----------------------------------------------------
+|1.0.0-beta.1 | 2022-05-01, 2023-04-01, 2023-11-15-preview (default)
```
Comment (bidisha-c) [inline sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md:31]: Newest API version 2024-11-15-preview?

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/src/AnalyzeConversationAuthoring.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+using Azure.Core;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.AI.Language.Conversations.Authoring;
+
+namespace Azure.AI.Language.Conversations.Authoring
+{
+    /// <summary> The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in &lt;a href=\"https://docs.microsoft.com/azure/cognitive-services/language-service/overview\"&gt;https://docs.microsoft.com/azure/cognitive-services/language-service/overview&lt;/a&gt;.0. </summary>
+    public partial class AnalyzeConversationAuthoring
```
Comment (bidisha-c) [inline sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/src/AnalyzeConversationAuthoring.cs:17]: Looks like an empty class. Do you plan to update?

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md (hunk 1 above)
Comment (GouriKA) [reply sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md:31]: Yes, this API version should be included in supported APIs

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+Uri endpoint = new Uri("https://myaccount.cognitiveservices.azure.com");
+AzureKeyCredential credential = new("your-api-key");
+AuthoringClient client = new AuthoringClient(endpoint, credential);
+AnalyzeConversationAuthoring authoringClient = client.GetAnalyzeConversationAuthoringClient();
+```
+
+#### Create a client using Azure Active Directory authentication
+
+You can also create a `AnalyzeConversationAuthoring` using Azure Active Directory (AAD) authentication. Your user or service principal must be assigned the "Cognitive Services Language Reader" role.
+Using the [DefaultAzureCredential] you can authenticate a service using Managed Identity or a service principal, authenticate as a developer working on an application, and more all without changing code.
```
Comment (GouriKA) [inline sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md:72]: change `and more all without changing code.` to `without changing code.`

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md (hunk 1 above)
Comment (amber-ccc) [reply sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/README.md]: Bidisha, thanks for your comments. Have updated now.

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/src/AnalyzeConversationAuthoring.cs (hunk 2 above)
Comment (amber-ccc) [reply sdk/cognitivelanguage/Azure.AI.Language.Conversations.Authoring/src/AnalyzeConversationAuthoring.cs]: I was thinking about adding specific constructors, now I see that it is not needed. Have deleted it now.

**Comment by amber-ccc** [comment]
> Change log is missing.

@bidisha-c Hi Bidisha, thanks for commenting! This Conversations authoring sdk is the first version sdk, so it might not need ChangeLog?

**Review by bidisha-c — APPROVED** [review]

**Pull request merged**